test/unit/spec/widgets/kbaseTabs-spec.js

```diff
@@ -8,13 +11,283 @@ define(['jquery', 'kbaseTabs'], ($, KBaseTabs) => {
         it('should render with minimal options', () => {
             const $host = $(document.createElement('div'));
             new KBaseTabs($host, {
-                tabs: [{
-                    tab: 'Foo',
-                    content: 'BAR'
-                }]
+                tabs: [
+                    {
+                        tab: 'Foo',
+                        content: 'BAR',
+                    },
+                    {
+                        tab: 'Baz',
+                        content: 'Fuzz',
+                    },
+                ],
+            });
+            expect($host.text()).toContain('Foo');
+            expect($host.text()).toContain('BAR');
+            expect($host.text()).toContain('Baz');
+            expect($host.text()).not.toContain('Fuzz');
```

Comment:
Can you make these tests more specific so they differentiate between what's in the tab label vs what's in the tab content?

Reply:
Yes! There is actually a change after this which narrows the scope to the nav tabs and pane separately, not in the PR yet.